traffic_ops/v4-client/servercheck.go

```diff
@@ -32,12 +35,22 @@ func (to *Session) InsertServerCheckStatus(status tc.ServercheckRequestNullable)
 	return &resp, reqInf, nil
 }
 
-// GetServersChecks fetches check and meta information about servers from /servercheck.
-func (to *Session) GetServersChecks() ([]tc.GenericServerCheck, tc.Alerts, ReqInf, error) {
-	var response struct {
+// GetServerCheckWithParamHdr fetches the Delivery Service with the given ID.
```

Comment:
I think this should maybe read something like

> GetServersChecks fetches 'Server Checks' from Traffic Ops.

Reply:
oops.